package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "nrfconnect",
-    "version": "3.4.1",
+    "version": "3.4.0",
```

Comment:
Probably an increase would be better choice :)

Reply:
Yes, this was used for testing function after upgrading, will revert.